themes_cern/indico_themes_cern/templates/council.html

```diff
@@ -2,3 +2,28 @@
 
 {# Hide location #}
 {% set show_siblings_location = false %}
+
+{# Customize "day header" to add time information #}
+{% block day_header %}
+    <div class="day-header" style="width: 100%;">
+        <div class="day-title" data-anchor="{{ anchor }}">
+            {{ item.start_dt | format_date(format='EEEE, d MMMM', timezone=timezone) }}
+            at
+            {{ item.start_dt | format_time(format='short') }}
+        </div>
+        {% if days|length > 1 %}
```

Comment:
why not `{% if days`?

Reply:
It's code that was already there (just indented it) but I thought it was a generator. Indeed, it's already a list. So, yeah.